lib/rules/no-unused-class.js

```diff
@@ -63,13 +64,12 @@ export default {
 
         const styleFileAbsolutePath = path.resolve(dirName, styleFilePath);
 
-        /*
-           maps classNames with a boolean to mark as used in source
-         */
-        const classNamesMap = getStyleClasses(styleFileAbsolutePath);
-
         // this will be used to mark s.foo as used in MemberExpression
-        _.set(map, `${importName}.classes`, classNamesMap);
+        const classes = getStyleClasses(styleFileAbsolutePath);
+        const classesMap = classes && getClassesMap(classes, camelCase);
+
+        _.set(map, `${importName}.classes`, classes);
+        _.set(map, `${importName}.classesMap`, classesMap);
```

Comment:
Can you update the comment near line 43 with this new structure of map.
Also update the comment in the other file.